### DISCUSSION TITLE
How to loop the presentation?

### DISCUSSION THREAD

**Discussion by redblackeagleman** [slidevjs/slidev#245] [Q&A]
Hey there,

I have been playing with the slidev a little - and really like it so far, and need to prepare a presentation of specific things (each slide updates on a different day) that should run continuously on my bosses screen. Anyone knows if there is a way how to make the presentation loop back to the first slide when it reaches the end?

**Comment by antfu** [comment]
You can write a script to control the navigations as you want with https://sli.dev/custom/vue-context.html and https://sli.dev/custom/global-layers.html
